raxx_cowboy/lib/raxx/cowboy/handler.ex

```diff
@@ -59,10 +60,15 @@ defmodule Raxx.Cowboy.Handler do
     {:ok, body, _req} = parse_req_body(req, content_type)
 
     # {peer, req} = :cowboy_req.peer req
+    
+    scheme = case transport do
+      :ssl -> "https"
```

Comment:
My mistake it looks like you are quite right about the `{:ssl, :http}`. it's just the indentation and this is good to merge. can open a new issue about getting elli to support ssl and assume the correct port